lxd/device/nic_physical.go

```diff
@@ -48,6 +48,10 @@ func (d *nicPhysical) validateConfig(instConf instance.ConfigReader) error {
 
 // validateEnvironment checks the runtime environment for correctness.
 func (d *nicPhysical) validateEnvironment() error {
+	if d.inst.Type() == instancetype.VM && shared.IsTrue(d.inst.ExpandedConfig()["migration.stateful"]) {
```

Comment:
We tend to use the term "NIC" when referring to network interface devices, so perhaps these errors would be more consistent with the docs and instance device config using a format like "Physical NIC devices cannot..."?

Reply:
Yep, will update